.gitmodules

```diff
@@ -1,3 +1,7 @@
 [submodule "docs"]
 	path = docs
 	url = https://github.com/pine64/bl602-docs.git
+
+[submodule "components/security/mbedtls"]
+	path = components/security/mbedtls
+	url = git@github.com:ARMmbed/mbedtls.git
```

Comment:
Unnecessary SSH authentication. `https://github.com/ARMmbed/mbedtls`

Reply:
This also is updated in a commit about to be pushed :)